### DISCUSSION TITLE
Hydration error on layout

### DISCUSSION THREAD

**Discussion by TCGAUD** [vercel/next.js#46505] [Help]
Why am i getting hydration error

import NavBar_pop_up from './NavBar_pop_up'
import UserDropDown from './UserDropDown';
import Left_NavBar from './Left_NavBar';
import Footer from './Footer';
import 'bootstrap/dist/css/bootstrap.css'
import '../globals.css'

export const metadata = {
  title: 'Penguin Management System',
  description: 'Generated by Next.js_13',
}

export default function RootLayout({
  children,
}: {
  children: React.ReactNode
}) {
  return (
    <html lang="en">
      <div className="sb-nav-fixed">
        <nav className="sb-topnav navbar navbar-expand navbar-dark bg-dark">
          <NavBar_pop_up />
          <UserDropDown />
        </nav>
        <div id="layoutSidenav">
          <Left_NavBar />
          <div id="layoutSidenav_content">
            <main>
            <body>{children}</body>
            </main>
            <Footer />
          </div>
        </div>
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" crossOrigin="anonymous"></script>
      </div>
      
    </html>
  )
}

**Comment by TCGAUD** [comment]
This is in layout folder

**Comment by icyJoseph** [comment]
Hey!

Could you please use https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting, I am not sure I follow your code snippet.

Also, could you post screenshots of the errors? or paste some relevant info?
